circle.yml

```diff
@@ -548,11 +547,8 @@ jobs:
           name: Run Android unit tests
           command: make run-android-unit-test
       - run:
-          name: Build libmapbox-gl.so for arm-v7
-          command: make android-lib-arm-v7
-      - run:
-          name: Compile Core tests for arm-v7
-          command: make android-test-lib-arm-v7
```

Comment:
We don't need to compile core tests anymore, this was in place as we didn't have CI integration for it. Now with metrics build actually running them, we don't need to keep this step.